app/actions/navigation/index.js

```diff
@@ -91,20 +98,25 @@ export function resetToSelectServer(allowOtherServers) {
                             allowOtherServers,
                         },
                         options: {
+                            animations: {
+                                setRoot: {
+                                    waitForRender: true,
+                                },
+                            },
                             layout: {
-                                backgroundColor: theme.centerChannelBg,
-                                componentBackgroundColor: theme.centerChannelBg,
+                                backgroundColor: colorStyles.navigation.backgroundColor,
+                                componentBackgroundColor: colorStyles.navigation.backgroundColor,
```

Comment:
@migbot I think my latest commit should fix it. The background color should come from the container not navigation `colorStyles.container.backgroundColor`, if I understand the `layout` key correctly. Navigation is for the top bar. I probably should have named it topBar instead of navigation.